From 89ff3f388ab80380bcf467fcf6783aee3a63c5d6 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 19 Apr 2021 22:37:13 +0200
Subject: [PATCH] target/linux/generic: IR_IMON_RAW is not set

Fix:

    SoundGraph iMON Receiver (early raw IR models) (IR_IMON_RAW) [N/m/?] (NEW)
---
 target/linux/generic/config-5.10 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/generic/config-5.10 b/target/linux/generic/config-5.10
index 13226c1bd6..03839d36fc 100644
--- a/target/linux/generic/config-5.10
+++ b/target/linux/generic/config-5.10
@@ -2723,6 +2723,7 @@ CONFIG_IP_VS_MH_TAB_INDEX=10
 # CONFIG_IR_IGUANA is not set
 # CONFIG_IR_IMG is not set
 # CONFIG_IR_IMON is not set
+# CONFIG_IR_IMON_RAW is not set
 # CONFIG_IR_JVC_DECODER is not set
 # CONFIG_IR_LIRC_CODEC is not set
 # CONFIG_IR_MCEUSB is not set
-- 
2.25.1

